acc_test/resource_cluster_test.go

```diff
@@ -231,6 +233,33 @@ func TestAccClusterCustomVPC(t *testing.T) {
 	})
 }
 
+func TestAccKafkaZookeeperContactPoints(t *testing.T) {
```

Comment:
[required] lets see a unit test with an odd number of AZs for good measure

Reply:
built a second test on Cassandra with 3 racks 1 node per rack. Running it now.